runtimes/protocol/lsp.ts

```diff
@@ -85,6 +85,10 @@ export interface AWSInitializationOptions {
      * Additional clientInfo to extend or override default data passed by LSP Client.
      */
     clientInfo?: ExtendedClientInfo
+    /**
+     * Client data directory, used to store LSP server-specific data.
```

Comment:
Can you clarify more in documentation the purpose of this field, and what are the expectation for Servers and Clients about using it?